### DISCUSSION TITLE
Virtual Worlds and Simulations meeting agenda for 2023-02-15

### DISCUSSION THREAD

**Discussion by OBWANDO** [Open-MV/fig-virtualworldsim#3] [Meetings]
## Meeting Details

- **Date/Time:** February 15th, 2023 @ Wednesday 12:00 PM ET / 17:00 UTC / 6:00 PM CET
- **Location:** Link should be posted in the `#Virtual-world-sim` text and voice channel on Discord at least 5 days before the meeting.
- **Moderator:** @LaylBongers 
- **Note Taker** @LaylBongers 

The [FIG Virtual Worlds and Simulations Meetings](https://github.com/Open-MV/fig-virtualworldsim/tree/main/meetings) repo contains the history past meetings, including a link to the agenda, recording, notes, and resources.

## Meeting Agenda
Database Feature triage

**Discuss agenda from proposed topics**

## Outcomes from Discussion topics

- Industrial users have an interest in saving money using large scale integration tests of their plants, to prototype programs and overall production flow before making large investments or pushing out updates
- Maybe early prototypes don't need to be broadly compatible if they exist primarily for testing the development of formats

## Action Items

- Priority item on "metaverse viewer project" (Tracking issue #4), we need to start looking at this and exploring what existing projects we can make use of too to reduce our workload in this prototyping stage.
- Break down the "Avatar-as-Metaobject" task into its dependencies, and derived features, so we can fit it under a larger "User Avatars" epic.
- Add tracking issues for database items that we consider at this point to be obvious high priorities to be handled soon, and expand on the two already added.

**Create actionable items from proposed topics**

## Items for next week's meeting
**List items that need to be discussed next week**

## Open Discussion Items

List any additional items below!

**Comment by LaylBongers** [comment]
Proposed agenda topics:
- triaging database entries, we've got a lot already covering stuff we may want to do
- discussion on metaverse scripting, including platform limitations on consoles and phones
- update on daicon and dacti formats work, if there's feedback or notes
- actionable plan for a roadmap to replicated user avatars prototype

Some of last week topics carried over with notes:
- "file-as-object, "self-descriptive object"
- revisiting protocol middleware, any new feedback and research?
- interfacing with third party virtual worlds (suggested TeleportVR client object)

**Comment by LaylBongers** [comment]
## Outcomes from Discussion topics

- Industrial users have an interest in saving money using large scale integration tests of their plants, to prototype programs and overall production flow before making large investments or pushing out updates
- Maybe early prototypes don't need to be broadly compatible if they exist primarily for testing the development of formats

## Action Items

- Priority item on "metaverse viewer project" (Tracking issue #4), we need to start looking at this and exploring what existing projects we can make use of too to reduce our workload in this prototyping stage.
- Break down the "Avatar-as-Metaobject" task into its dependencies, and derived features, so we can fit it under a larger "User Avatars" epic.
- Add tracking issues for database items that we consider at this point to be obvious high priorities to be handled soon, and expand on the two already added.
